madmin/templates/status.html

```diff
@@ -0,0 +1,55 @@
+{% extends "base.html" %}
+
+{% block header %}
+{% endblock %}
+
+{% block scripts %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.2/moment.min.js"></script>
+<script>
+    function setGrid(tableGridHtmlId, gridData) {
+        $(tableGridHtmlId).DataTable({
+            "data": gridData,
+            "columns": [
+                { data: 'name', title: 'Name' },
+                { data: 'lastUpdate', title: 'Lastupdate' },
+                { data: 'lat', title: 'Position' }
+            ],
+            "columnDefs": [
+                {
+                    "targets": [1],
+                    "render": function (data, type, row) {
+                        return moment.unix(data).format('YYYY-MM-DD HH:mm:ss');
```

Comment:
maybe prefix with a "(x minutes ago)" for better readability,

Reply:
Agree on this